### PR TITLE
Add changes from Twig-View 2.5.1

### PR DESCRIPTION
**Pull request by stevenubc**
Pull in changes from the Twig View library to support Twig 3.

### CODE REVIEW

File: src/Twig.php [hunk 1]
```diff
@@ -137,11 +142,11 @@ public function render(ResponseInterface $response, $template, $data = [])
      * Create a loader with the given path
      *
      * @param array $paths
-     * @return \Twig_Loader_Filesystem
+     * @return \Twig\Loader\FilesystemLoader
      */
     public function createFilesystemLoader(array $paths)
     {
-        $loader = new \Twig_Loader_Filesystem();
+        $loader = new \Twig\Loader\FilesystemLoader();
```
Comment (stevenubc) [inline src/Twig.php:149]: Available in Twig 1. See https://github.com/twigphp/Twig/blob/1.x/lib/Twig/Loader/Filesystem.php#L2C1-L3C34

File: src/TwigExtension.php [hunk 2]
```diff
@@ -34,10 +36,11 @@ public function getName()
     public function getFunctions()
     {
         return [
-            new \Twig_SimpleFunction('path_for', array($this, 'pathFor')),
-            new \Twig_SimpleFunction('base_url', array($this, 'baseUrl')),
-            new \Twig_SimpleFunction('is_current_path', array($this, 'isCurrentPath')),
-            new \Twig_SimpleFunction('current_path', array($this, 'currentPath')),
+            new \Twig\TwigFunction('path_for', array($this, 'pathFor')),
```
Comment (stevenubc) [inline src/TwigExtension.php:39]: Available in Twig 1: See https://github.com/twigphp/Twig/blob/1.x/src/TwigFunction.php#L23

File: src/Twig.php [hunk 3]
```diff
@@ -100,7 +105,7 @@ public function fetchBlock($template, $block, $data = [])
     {
         $data = array_merge($this->defaultVariables, $data);
 
-        return $this->environment->loadTemplate($template)->renderBlock($block, $data);
+        return $this->environment->load($template)->renderBlock($block, $data);
```
Comment (stevenubc) [inline src/Twig.php:108]: Available in Twig 1. See https://github.com/twigphp/Twig/blob/1.x/src/Environment.php#L413

**Comment by schuyberg** [comment]
Looks good. Let's be sure to add a version tag to this.

**Comment by schuyberg** [comment]
We should also update the README to reflect the updated version.

**Comment by stevenubc** [comment]
Thanks Schuyler, updated the README with Twig 3

**Pull request merged**